### PR TITLE
catalog version

### PR DESCRIPTION
**Pull request by ialberquilla**
## Description

Updates catalog version

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] Documentation

### CODE REVIEW

**Comment by r-marques** [comment]
@dancespiele can you help  with this pr?

**Comment by ialberquilla** [comment]
Fixed some compilation errors, but there are more looks like now `const { subscription  } = Catalog.useNevermined()` is not part of the catalog anymore

**Comment by r-marques** [comment]
> 



> Fixed some compilation errors, but there are more looks like now `const { subscription } = Catalog.useNevermined()` is not part of the catalog anymore

No. It was renamed to `nfts` I think:
```ts
const { nfts } = Catalog.useNevermined()

// instead of `buySubscription`
const response = await nfts.access(...)
```

https://docs.nevermined.io/docs/catalog/example#buyasset

**Comment by ialberquilla** [comment]
Build errors solved, ready to review now

File: client/pages/_app.tsx [hunk 1]
```diff
@@ -17,35 +17,33 @@ import {
...
-  nodeUri,
+  web3ProviderUri: nodeUri,
   marketplaceUri,
-  gatewayUri,
+  neverminedNodeUri: gatewayUri,
   faucetUri,
-  gatewayAddress,
-  secretStoreUri,
+  newGateway: true,
+  neverminedNodeAddress: gatewayAddress,
```
Comment (dancespiele) [inline client/pages/_app.tsx:38]: this should be called the same than the config prop `neverminedNodeAddress`

File: client/src/shared/graphql.ts [hunk 2]
```diff
@@ -36,7 +35,7 @@ export interface RegisteredAsset {
 
 export const loadPublishedEvent = async (
   asset: string,
-  provider: MetaMask.MetamaskProvider
+  provider: any
```
Comment (dancespiele) [inline client/src/shared/graphql.ts:38]: the same than above

File: client/src/shared/graphql.ts [hunk 3]
```diff
@@ -152,7 +151,7 @@ export const loadAssetProvenance = async (
 
 export const getUserSubscription = async (
   sdk: Nevermined,
-  provider: MetaMask.MetamaskProvider,
+  provider: any,
```
Comment (dancespiele) [inline client/src/shared/graphql.ts:154]: the same about the type

File: client/src/components/+token-name/token-name.tsx [hunk 4]
```diff
@@ -18,11 +18,11 @@ class TokenNameGetter {
     }
   }
 
-  static async getSymbol(provider: MetaMask.MetamaskProvider, address?: string) {
+  static async getSymbol(provider: any, address?: string) {
```
[the remark this answers is not in the document]
Comment (ialberquilla) [reply client/src/components/+token-name/token-name.tsx:21]: Getting this error using this type 

`Argument of type 'import("/home/ivan/Repo/defi-marketplace/client/node_modules/@ethersproject/abstract-provider/lib/index").Provider' is not assignable to parameter of type 'import("/home/ivan/Repo/defi-marketplace/client/node_modules/@wagmi/core/dist/declarations/src/types/index").Provider'.`

File: client/src/shared/graphql.ts (hunk 3 above)
Comment (ialberquilla) [reply client/src/shared/graphql.ts]: Fixed

File: client/pages/_app.tsx (hunk 1 above)
Comment (ialberquilla) [reply client/pages/_app.tsx]: Fixed

**Review by dancespiele — APPROVED** [review]

**Pull request merged**